### PR TITLE
feat: wish-list page

### PR DESCRIPTION
**Pull request by TheAmlanSaikia**

### CODE REVIEW

**Review by Aditi0505 — APPROVED** [review]
Looks good to me.
You can add description and screenshots for better visibility.

File: Wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+.navbar {
+  grid-area: navbar;
+  min-height: 11vh;
+  background-color: var(--pure-white);
+  box-shadow: var(--box-shadow-dark);
+  padding: var(--value-2) var(--value-2);
+  top: 0;
+  z-index: 10;
+  width: 100vw;
```
Comment (anupkgurung) [inline Wishlist/wishlist.css:33]: why 10 z-index ?

**Review by anupkgurung — APPROVED** [review]
LGTM.

File: Wishlist/wishlist.css (hunk 1 above)
Comment (TheAmlanSaikia) [reply Wishlist/wishlist.css:33]: Ohh, I have set my highest z-index as 10, which is the navbar. Other z-index values are between 0 to 10.

**Pull request merged**